src/utils/ReduxUtils.ts

```diff
@@ -38,9 +44,12 @@ export interface IReduxAction<T> extends Redux.Action {
 
 export interface IReduxActionPayload {
   id?: string;
-  filterText?: string;
+  ids?: string[];
   facet?: string;
   facetRow?: IFacet;
+  filterText?: string;
+  pageNb?: number;
```

Comment:
Having a global IReduxActionPayload interface is clearly not where I would like to see this project go. I'm sorry, but each action should have it own "Payload" interface, not use a global one for the entire project. IMO, the "members example" is a quite good example on how we could strongly type each actions and their payload.